docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -349,6 +350,9 @@ EOF
 ----
 +
 <1> Select an existing storage class in your {OpenShiftShort} cluster.
+<2> Adjust the dnsmasq server entries as needed. In order to access MetalLB resources on OCP control plane,
+including messaging and database services, you should provide IP addresses of OCP master nodes.
+For external, or other resources on the intranet, you may want to provide additional IP addresses for datacentre DNS resolvers.
```

Comment:
192.168.122.1 should be the one which can resolve any other names, either from external (internet) or from services in the datacentre.